systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -67,6 +70,38 @@ public class CruiseControlST extends AbstractST {
 
     private final String namespace = testSuiteNamespaceManager.getMapOfAdditionalNamespaces().get(CruiseControlST.class.getSimpleName()).stream().findFirst().get();
 
+    @IsolatedTest
+    void testJvmAndResources(ExtensionContext extensionContext) {
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaWithCruiseControl(clusterName, 3, 3)
+            .editMetadata()
+                .withNamespace(namespace)
+            .endMetadata()
+            .editOrNewSpec()
+                .editCruiseControl()
+                    .withResources(new ResourceRequirementsBuilder()
+                        .addToLimits("memory", new Quantity("1.5Gi"))
+                        .addToLimits("cpu", new Quantity("1"))
+                        .addToRequests("memory", new Quantity("1Gi"))
+                        .addToRequests("cpu", new Quantity("0.8"))
+                        .build())
+                    .withNewJvmOptions()
+                        .withXmx("1G")
+                        .withXms("256M")
+                        .withXx(Maps.of("UseG1GC", "true"))
+                    .endJvmOptions()
+                .endCruiseControl()
+            .endSpec()
+            .build());
+
+        String ccPodName = kubeClient().listPodsByPrefixInName(namespace, CruiseControlResources.deploymentName(clusterName)).get(0).getMetadata().getName();
+        assertResources(namespace, ccPodName, "cruise-control",
+                "1.5Gi", "1", "1Gi", "800m");
+        assertExpectedJavaOpts(namespace, ccPodName, "cruise-control",
+                "-Xmx1G", "-Xms256M", "-XX:+UseG1GC");
+    }
+
```

Comment:
The test itself is good, but I think you can add it to another test, which is already written. My point is that you will wait for example 5 minutes for Kafka and CC will be up and then you just check that resources are correct - which is little bit small check for the extra time.

Reply:
Yup, already added it into `testAutoCreationOfCruiseControlTopicsWithResources` (and changed name). 
I think it won't affect original test in any way. Agree?